games/data/bopl-battle.yml

```diff
@@ -28,5 +38,13 @@ thunderstore:
       name: "Modpacks"
       requireCategories:
         - "modpacks"
-  wikiUrl: "https://github.com/commandblox/Splotch/wiki"
-  discordUrl: "https://discord.gg/official-bopl-battle-modding-comunity-1175164882388275310"
+    maps:
+      name: "Maps"
+      requireCategories:
+        - "maps"
+    abilities:
+      name: "Abilities"
+      requireCategories:
+        - "abilities"
+wikiUrl: "https://borpl.site/wiki"
+discordUrl: "https://discord.gg/official-bopl-battle-modding-comunity-1175164882388275310"
```

Comment:
CI validation fails because these aren't indented (and as such they end up to the wrong location in the data)